gettsim/data/e_st_abzuege.yaml

```
@@ -346,12 +354,24 @@ altenth:
     2019-01-01:
       value: 836
       note:
+    2020-01-01:
+      value: 760
+      note:
```

```suggestion
      note: Artikel 1 G. v. 05.07.2004 BGBl. I S. 1427.
```

If you add a paramter/date/value, please also fill the note.

If 24a EStG S. 5 adds all the parameters for a long period, always cite the BGBl that defines these parameters.

also following instances.